feature/staticroutes/staticroutes.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
the directory should be feature/staticroute/. We don't use plural for the feature name.

Reply:
Changed Done!